### DISCUSSION TITLE
Some noob questions

### DISCUSSION THREAD

**Discussion by jupppo** [johanmeijer/grott#218] [Q&A]
Hello,

I managed to install grott on a seperat debian instance. I now have a question:
Do I need to install MQTT Server as well to send over data to home assistant?
Can I check somehow whether grott receives data? I changed the server adress on my Sine S Stick, but I dont knwo whether grott gets data.

Best 
Thomas

**Comment by johanmeijer** [comment]
Yes you need mqtt to be installed as well.

I suppose you installed Grott as a linux service? Then you can look at the Linux syslog if Grott Is working and what it is doing.

I usually use the journalctl -u grott or journalctl -u grott -e (to limit the output) commands

**Comment by jupppo** [comment]
Thanks a Lot, seems to bei working now. 
Still struggling how to integrate the Data Info Home Assistent.
So you know what include Statement to you yaml File needs to ne added to configuration.yaml?

**Comment by johanmeijer** [comment]
There are sample yaml files available in the examples folder.

**Comment by jupppo** [comment]
Yes, I have seen than. But I am struggling how to use and include.

**Comment by johanmeijer** [comment]
Ok. Now I hope somebody else can help. I am running HA since 2 weeks in a container since 2 weeks but haven't done anything with it yet.

**Comment by jupppo** [comment]
Thanks Johan,

I finally managed by splitting the sections and putting it under the sensor: tag of my configuration.yaml
Could it be that not everything is omplemented yet? I am missing the actual ac load (from grid) in the data.
I tested a update frequenzy of 2 sec., which runs great. Dont care about growatt servers anymore, when I get all the metrics locally. 

Thomas
